auditlog-fhir-server/src/main/java/ca/uhn/fhir/jpa/starter/MyConsentService.java

```diff
@@ -91,7 +91,18 @@ private void createAuditEvent(RequestDetails theRequestDetails, boolean wasSucce
     auditEvent.addExtension(basedOnExtension);
 
     AuditEvent.AuditEventAgentComponent agentComponent = new AuditEvent.AuditEventAgentComponent();
-    agentComponent.setRequestor(true);
+    agentComponent.setRequestor(false);
+
+    //set who
+    try {
+      agentComponent.setWho(new Reference(retrieveSubjectIdOrPatientId(theRequestDetails)));
+      CodeableConcept roleType = new CodeableConcept();
+      roleType.addCoding(new Coding("http://terminology.hl7.org/CodeSystem/v3-RoleClass", "PAT", "patient"));
+      agentComponent.addRole(roleType);
+    } catch (Exception e) {
+      System.out.println(e);
```

Comment:
Is there an option for a logger? Because I think the Println statement won't be noticed from inside the docker container. Thus if there ever occurs an exception we won't be able to find the problem.

Reply:
there most definitely is. adding a logger right now, println is super ugly anyways :)